scripts/README.md

```diff
@@ -17,7 +17,34 @@
   * Commit changes and push to origin
   * Open github to submit the PR
 
-## Update sitemap
+### Extracting texts from js code:
+Texts that used in js code would automatically be extracted during the translation update process (3rd step above: Updating the source file).
 
-* List of paths to include in `sitemap.xml` is here: [sitemap.js](sitemap.js)
+In order to make sure that the extractor code won't miss any string to push to translations, please:
+
+1. Refactor the code so that the first argument passed to `localize()` method, be a string literal.
+    i.e.
+    ```js
+    const text = localize(is_started ? 'Sell at market' : 'Sell');
+    ```
+    could change to:
```

Comment:
I think `would` instead of `could`, as it's not an optional change but a required change